superset-frontend/src/components/TimezoneSelector/index.tsx

```diff
@@ -23,7 +23,7 @@ import moment from 'moment-timezone';
 import { NativeGraySelect as Select } from 'src/components/Select';
 
 const DEFAULT_TIMEZONE = 'GMT Standard Time';
-const MIN_SELECT_WIDTH = '375px';
+const MIN_SELECT_WIDTH = '400px';
```

Comment:
we should be able to overwrite this in our component instead of the "base" component.